core/renderers/common/constants.js

```diff
@@ -401,6 +400,14 @@ Blockly.blockRendering.ConstantProvider = function() {
    */
   this.randomIdentifier = String(Math.random()).substring(2);
 
+  /**
+   * The defs tag that contains all filters and patterns for this Blockly
+   * instance.
+   * @type {SVGElement}
+   * @private
+   */
+  this.defs = null;
```

Comment:
I think we are keeping `_` on class properties.

Reply:
Done.